gdal/doc/source/development/rfc/rfc51_rasterio_resampling_progress.rst

```diff
@@ -105,6 +105,7 @@ The following resampling algorithms are available :
        /*! Cubic B-Spline Approximation (4x4 kernel) */    GRIORA_CubicSpline = 3,
        /*! Lanczos windowed sinc interpolation (6x6 kernel) */ GRIORA_Lanczos = 4,
        /*! Average */                                      GRIORA_Average = 5,
+       /*! Average */                                      GRIORA_AverageQuadratic = 13,
```

Comment:
this addition can be removed. The RFC represents the state of the code at the time it was written